nexus/tests/integration_tests/unauthorized_coverage.rs

```diff
@@ -138,7 +138,20 @@ fn test_unauthorized_coverage() {
     let expected_uncovered_endpoints =
         std::fs::read_to_string("tests/output/uncovered-authz-endpoints.txt")
             .expect("failed to load file of allowed uncovered endpoints");
-    assert_eq!(expected_uncovered_endpoints, uncovered_endpoints);
+    let mut unexpected_uncovered_endpoints = "These endpoints were expected to be covered by the unauthorized_coverage test but were not:\n".to_string();
+    let mut has_uncovered_endpoints = false;
+    for endpoint in uncovered_endpoints.lines() {
+        if !expected_uncovered_endpoints.contains(endpoint) {
+            unexpected_uncovered_endpoints
+                .push_str(&format!("\t{}\n", endpoint));
+            has_uncovered_endpoints = true;
+        }
+    }
+    assert_eq!(
+        has_uncovered_endpoints, false,
+        "{}\nMake sure you've added a test for this endpoint in unauthorized.rs.",
+        unexpected_uncovered_endpoints
+    )
```

Comment:
I have been on the other end of debugging this test one too many times. If you click through to the comment it's helpful sort of in explaining what's happening, but the test output most of the time is entirely illegible. This just updates the test output to tell you a list of things you need to write tests for instead of an inscrutable mismatch of file contents.